scripts/generate/templates/react.template.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Extend Shopify Checkout with a custom Post Purchase user experience
+ * This Shopify Checkout template provides two extension points:
+ *  1. Inquiry - Called first, during the payment step.
```

Comment:
I think it makes sense to have that kind of documentation here even if it's partly duplicated because it's in close proximity to where developers are going to need it. 👍 

Wonder if we can / should link to a readme with more info from here (as soon as we have that)?

Should we mention the `payment` step? We might be calling that at a different time in the future. That might esp. be true for payment methods like shop pay.

Reply:
Good suggestion.  I'll remove the reference to `payment`.